app/assets/Components/ListItem/ListItem.js

```diff
@@ -16,7 +16,50 @@ class ListItem extends Component {
       day: '',
       dailyWeather: {},
       weatherFetched: false,
-      showInput: false
+      showInput: false,
+      expanded: false
+    }
+  }
+
+  toggleLowerCard() {
+    const newExpandedStatus = !this.state.expanded
+    this.setState({
+      expanded: newExpandedStatus
+    })
+  }
+
+  displayLowerCard(completedClass, completedText) {
```

Comment:
I think you could break this into its own component - since you already have it in a named function